sql/core/src/test/scala/org/apache/spark/sql/execution/command/v2/ShowTablesSuite.scala

```diff
@@ -19,21 +19,15 @@ package org.apache.spark.sql.execution.command.v2
 
 import org.apache.spark.sql.{AnalysisException, Row}
 import org.apache.spark.sql.execution.command
-import org.apache.spark.sql.types.{StringType, StructType}
 
 /**
  * The class contains tests for the `SHOW TABLES` command to check V2 table catalogs.
  */
 class ShowTablesSuite extends command.ShowTablesSuiteBase with CommandSuiteBase {
   override def defaultNamespace: Seq[String] = Nil
-  override def showSchema: StructType = {
-    new StructType()
-      .add("namespace", StringType, nullable = false)
-      .add("tableName", StringType, nullable = false)
-  }
   override def getRows(showRows: Seq[ShowRow]): Seq[Row] = {
```

Comment:
do we still need this? All the implementations are the same.

Reply:
I have put `getRows `in parent.